src/games/strategy/engine/data/ChangeFactory.java

```diff
@@ -105,6 +105,7 @@ public static Change changeResourcesChange(final PlayerID player, final Resource
   }
 
   /** TODO: unused code, kept per @veqryn */
+  @SuppressWarnings("unused")
```

Comment:
remove the `@SuppressWarnings` unless this is actually used. Better to have the compiler error, otherwise what this annotationo is saying is that compile time checks are insufficient to show that this code *actually* is used (in which case if you give it the  axe, code will still compile, but bomb at runtime)